### ISSUE TITLE
uib-tooltip-template error

### ISSUE THREAD

**Issue by mjw56** [angular-ui/bootstrap#4953]
Hi,

I am using ui-bootstrap 0.14.3 and angular 1.4.1.

I am trying to setup a tooltip using an ng-template:

```html
<script type="text/ng-template" id="myTooltipTemplate.html">
  <span>test</span>
</script>
```

and then in the same HTML file try to invoke it:

```html
<tr tooltip-placement="top" uib-tooltip-template="'myTooltipTemplate.html'">
  <td>test</td>
  <td>test</td>
</tr>
```

however, I'm seeing the following error in the console:

```js
Error: Syntax error, unrecognized expression: <span>test</span>
```

I do see the tooltip dialog, however it is empty.

**Comment by wesleycho** [comment]
Please post a reproduction using Plunker.

Just a note, you should use angular 1.4.7.

**Comment by mjw56** [comment]
Hey @wesleycho, sure I can do that.

Is there a base Plunker I can go off of?

**Comment by wesleycho** [comment]
In the docs page, click the Edit in Plunker link for any particular component.

**Comment by mjw56** [comment]
Thanks, found the issue: jQuery 1.9.1

[Reproducible Plunk](http://plnkr.co/edit/OMjjTPGGQeEZfqAOF5SX?p=preview)

[Fix with jQuery 1.11.3](http://plnkr.co/edit/fAWERqnuqNyretGpGuUu?p=preview)
